scripts/verify-cognito-lambda-triggers.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Usage
+#   smoketest to verify the Cognito pool retains its Lambda triggers
+
+# Arguments
+#   - $1 - the environment [dev, stg, prod, exp1, exp1, etc]
+
+( ! command -v jq > /dev/null ) && echo "jq must be installed on your machine." && exit 1
+[ -z "${1}" ] && echo "ERROR: The env to run smoketest is missing from the \$1 argument.  An example value of this includes [dev, stg, prod... ]" && exit 1
+
+ENV="${1}"
+
+cognito_user_pool_name="efcms-${ENV}"
+
+user_pool_has_both_lambda_triggers=$(aws cognito-idp list-user-pools --max-results=60 | jq '.UserPools[] | select(.Name=="'"${cognito_user_pool_name}"'") | .LambdaConfig | .PostConfirmation and .PostAuthentication')
+
+if [ "${user_pool_has_both_lambda_triggers}" != 'true' ]; then
+  echo "ERROR: Cognito user pool ${cognito_user_pool_name} is missing either the PostConfirmation or PostAuthentication Lambda trigger"
+  exit 1
```

Comment:
Is exiting the right move here? If for some reason we get to this point, wouldn't we want to create triggers for the current color which would then be replaced during `switch-colorc`?

Reply:
Hmmm.  I think I rather fail than self-heal.  I want the correct behavior to be in Terraform, but we may never know if we don't have the correct behavior if this script self-heals a bug.

Reply:
I just wonder what is different then I guess. If we somehow don't have the triggers, then we have put ourselves in the exact situation that's being corrected right? An environment that was up, but we've broken Cognito? And it's checking for something that wasn't created by terraform in this case, but by a js script that was run in a previous run of the workflow.

Reply:
Correct, we'll be back in the same situation.  But this time we'll have an explicit error to help us understand what is wrong.  This is instead of us not knowing something is amiss.